src/servsock.js

```diff
@@ -0,0 +1,53 @@
+var fs = require('fs');
+var net = require('net');
+
+export default class ServiceSocket {
```

Comment:
I have thus far been unable to break anything. Sending SIGKILL to the client leaves orphaned children (the tail on the log files) but doesn't affect the service at all. Sending SIGTERM to the service after SIGKILL'ing a client shuts down the service with no errors.